rvv-intrinsic-rfc.md

```diff
@@ -348,70 +348,83 @@ vmerge.vvm vd, vs2, vs1, v0:
 | ------- | -------------------- | ------------------------- | ---------------------------------------
-| No      | No                   | N/A                       | `vmv_v_x_<ty>(src, vl)`
-| No      | Yes                  | N/A                       | `vmv_v_x_<ty>_tu(dest, src, vl)`
-
-For multiply-add intrinsics:
-| Masked? | Needs tail preserved | Needs maskedoff preserved | Intrinsic
-| ------- | -------------------- | ------------------------- | -------------------------------------------------------------------
-| No      | No                   | N/A                       | `vmacc_vv_<ty>(vd, vs1, vs2, vl)`
-| No      | Yes                  | N/A                       | `vmacc_vv_<ty>_tu(vd, vs1, vs2, vl)`
-| Yes     | No                   | No                        | No support.
-| Yes     | No                   | Yes                       | No support.
-| Yes     | Yes                  | Yes                       | No support.
-| Yes     | Yes                  | No                        | No support. Tail undisturbed and maskedoff agnostic is likely rare.
-
-If the type of the destination is mask type or scalar type, it is always tail-agnostic. The intrinsics are
-* Vector Integer Comparison Instructions
-* Vector Floating-Point Compare Instructions
-* Vector Floating-Point Classify Instruction
-* vmadc.vvm/vmadc.vxm/vmadc.vim
-* vmadc.vv/vmadc.vx/vmadc.vi
-* vmsbc.vvm/vmsbc.vxm
-* vmsbc.vv/vmsbc.vx
-* vmsbf.m
-* vmsif.m
-* vmsof.m
-* vcpop.m
-* vmv.x.s
-* vfmv.f.s
+| No      | No                   | N/A                       | `vmv_v_x_<ty>_ta(src, vl)` and `vmv_v_x_<ty>(src, vl)`
+| No      | Yes                  | N/A                       | `vmv_v_x_<ty>_tu(undisturbed, src, vl)`
+
+For multiply-add intrinsics or other intrinsics with the destination argument which may use tail and mask value.
+Masked?  | TU? | MU? | Intrinsic
+--- |--- |--- |---
+No  | No | N/A| `vmacc_vv_i8m1_ta(vd, vs1, vs2, vl)`
+No  | Yes| N/A| `vmacc_vv_i8m1_tu(vd, vs1, vs2, vl)` and `vmacc_vv_i8m1(vd, vs1, vs2, vl)`
+Yes | No | No | `vmacc_vv_i8m1_tama(mask, vd, vs1, vs2, vl)`
+Yes | No | Yes| `vmacc_vv_i8m1_tamu(mask, vd, vs1, vs2, vl)`
+Yes | Yes| No | `vmacc_vv_i8m1_tuma(mask, vd, vs1, vs2, vl)`
+Yes | Yes| Yes | `vmacc_vv_i8m1_tumu(mask, vd, vs1, vs2, vl)` and `vmacc_vv_i8m1_m(mask, vd, vs1, vs2, vl)`
+
+For reduction intrinsics which may only use tail value.
+
+Masked?  | TU? | MU? | Intrinsic
+--- |--- |--- |---
+No  | No | N/A| `redsum_vs_i8m1_ta(vs1, vs2, vl)`
+No  | Ye | N/A| `redsum_vs_i8m1_tu(vd, vs1, vs2, vl)` and `redsum_vs_i8m1(vd, vs1, vs2, vl)`
+Yes | No | No | `redsum_vs_i8m1_tama(mask, vs1, vs2, vl)`
+Yes | No | Yes| `redsum_vv_i8m1_m(mask, vd, vs1, vs2, vl)` if vd is `vundefined()`
+Yes | Yes| No | `redsum_vs_i8m1_tuma(mask, vd, vs1, vs2, vl)`
+Yes | Yes| Yes| `redsum_vs_i8m1_m(mask, vd, vs1, vs2, vl)`
+
+There are some special cases list as below:
+
+1. vadc/vsbc support `_ta` and _`tu`.
+2. vmadc/vmsbc support `_ta`.
```

Comment:
I think that special cases (2), (3), (5), and (6) could all omit `ta`, since they are always tail agnostic. It is true that some of these names collide with the legacy API, but I don't think any of these collisions cause incompatibilities.